sdk/template/.content/src/GlobalSuppressions.cs

```diff
@@ -3,4 +3,5 @@
 
 using System.Diagnostics.CodeAnalysis;
 
-[assembly: SuppressMessage("Usage", "AZC0002:DO ensure all service methods, both asynchronous and synchronous, take an optional CancellationToken parameter called cancellationToken.", Justification = "CancellationToken can be passed through RequestOptions")]
\ No newline at end of file
+// add global suppression here
+// [assembly: SuppressMessage(category, checkId)]
```

Comment:
Do we need this line?  Please remove these comments.  Ideally the template would indicate that we don't suppress warnings in our libraries.

Reply:
Actually, would it be possible to delete this file entirely, if there are no suppressions needed anymore?  It is a goal for our libraries that GA libraries don't ship with a GlobalSuppressions file - this should be reflected in the template as well.  Thanks!

Reply:
@annelo-msft Got it. Previously I thought we'd better to leave a place holder.